<launch>
  <arg name="INPUT" default="/camera/depth_registered/points" />
  <node pkg="nodelet" type="nodelet" name="manager"
        args="manager"
        output="screen"/>
  <node pkg="nodelet" type="nodelet" name="normal_estimate"
        args="load pcl/NormalEstimationOMP manager"
        clear_params="true">
    <remap from="~input" to="$(arg INPUT)" />
    <!-- <remap from="~output" to="/normal_concat/input" /> -->
    <rosparam>
      radius_search: 0
      k_search: 10
      # 0, => ANN, 1 => FLANN, 2 => Organized
      spatial_locator: 2
    </rosparam>
  </node>
  <node pkg="nodelet" type="nodelet" name="normal_concat"
        args="load jsk_pcl/NormalConcatenater manager"
        clear_params="true">
    <remap from="~input" to="$(arg INPUT)"/>
    <remap from="~normal" to="/normal_estimate/output"/>
  </node>
</launch>
